configure/jukebox.md

```diff
@@ -0,0 +1,80 @@
+---
+layout: docs
+title: Configure Jukebox
+permalink: /docs/configure/jukebox/
+---
+
+### Jukebox Properties
+
+Jukebox might not always work out-of-the-box and may require some additional tweaking. If you get no sound output while trying to play via the Jukebox, you might need to tweak the audio device being picked up by Java sound. You can run the folowing Java program to get a list of all the audio devices in your system:
+
+<details>
+    <summary>Source Code
+    </summary>
+<div markdown="1">
```

Comment:
Why do you have html code here ?
Please remove it

Reply:
It looked better if the code was expandable on demand, just for readability. 

![image](https://user-images.githubusercontent.com/584253/35404356-b07294de-0228-11e8-892b-e569280e8091.png)

(Removed in next commit)